core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.spec.ts

```diff
@@ -1751,6 +1751,51 @@ describe('Contributions and review component', () => {
 
     describe('when navigating to review tab', () => {
       it('should get in-review translation suggestions', fakeAsync(() => {
```

Comment:
@Ash-2k3 Add a test with a list of multiple strings?

Reply:
Done, Thanks!